### ISSUE TITLE
as a Citizen, I want to comment on issues on my mobile phone, so that I can be civically engaged anytime of the day

### ISSUE THREAD

**Issue by rskoenig** [drew-moore/civ-sandbox#4]

**Comment by drew-moore** [comment]
I'm tempted to close this: Mobile-first design is something we want to do everywhere, so this isn't a distinct user story in that respect, and apart from it it's too general.

**Comment by rskoenig** [comment]
Agreed that it's very general. In fact, there's not really any specific wireframes that it calls for. Nevertheless, we've experienced issues with iOS and touchscreen interfaces in the past so the acceptance criteria for this issue should be met before we call the story closed.

Acceptance criteria:
1. Unregistered users can register on an android device and on an iOS device.
2. Registered users can vote, comment and share on a android device and on an iOS device. 

Acceptance tests:
1. As an unregistered user, from an android phone, click on a link to an agenda item.
2. Click on the comment box.
3. You are prompted to signup.
4. Signup
5. Vote and comment on the agenda item.

1. As an unregistered user, from an iOS phone, click on a link to an agenda item.
2. Click on the comment box.
3. You are prompted to signup.
4. Signup
5. Vote and comment on the agenda item.

**Comment by rskoenig** [comment]
Browser Stack
